homeassistant/components/media_player/plex.py

```diff
@@ -0,0 +1,188 @@
+"""
+homeassistant.components.media_player.plex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Provides an interface to the Plex API
+
+Configuration:
+
+To use Plex add something like this to your configuration:
+
+media_player:
+  platform: plex
+  name: plex_server
+  user: plex
+  password: my_secure_password
+
+Variables:
+
+name
+*Required
+The name of the backend device (Under Plex Media Server > settings > server).
+
+user
+*Required
+The Plex username
+
+password
+*Required
+The Plex password
+"""
+
+import logging
+
+from homeassistant.components.media_player import (
+    MediaPlayerDevice, SUPPORT_PAUSE, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_NEXT_TRACK, MEDIA_TYPE_TVSHOW, MEDIA_TYPE_VIDEO)
+from homeassistant.const import (
+    STATE_IDLE, STATE_PLAYING, STATE_PAUSED, STATE_UNKNOWN)
+from plexapi.myplex import MyPlexUser
```

Comment:
One final thing, which will fix the build. To make this an optional platform, and hence `plexapi` dependency, can you delay the import? This is generally a python anti-pattern, but is done in this project to allow shipping all of the code together, but only installing the requirements, when the dependencies are installed.

Move:

```python
from plexapi.myplex import MyPlexUser
```

into `setup_platform()`. That will also have the benefit of fixing the build.

The changes look great! Thankyou for humoring me here.